src/components/StablecoinsTable.tsx

```diff
@@ -50,20 +50,18 @@ const StablecoinsTable = ({
     <Table
       variant="unstyled"
       my={8}
-      borderRadius="sm"
-      border={`1px solid ${textColor}`}
       bg="background.base"
       mb={8}
       minW="720px"
     >
-      <Thead bg="ednBackground" color="text200">
-        <Tr borderBottom={`1px solid ${textColor}`} mb="1px">
+      <Thead bg="background.highlight" color="text200">
```

Comment:
cc: @nloureiro okay switching `text200` (light: #666666, dark: #b2b2b2) to `body.medium` (light: gray.500 or #616161, dark: gray.400 or #8C8C8C)?

```suggestion
      <Thead bg="background.highlight" color="body.medium">
```

Reply:
> cc: @nloureiro okay switching `text200` (light: #666666, dark: #b2b2b2) to `body.medium` (light: gray.500 or #616161, dark: gray.400 or #8C8C8C)?

I'm very sorry, I overlooked `text200`. Thank you for pointing it out, I appreciate your help.